3.preprocess-features/3a.merge-features/3a.merge-cp-dp-features.py

```diff
@@ -0,0 +1,115 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# ### Import Libraries
+# 
+
+# In[1]:
+
+
+import pathlib
+import math
+import uuid
+import importlib
+import itertools
+
+import pandas as pd
+from pycytominer.cyto_utils import DeepProfiler_processing
+
+merge_utils = importlib.import_module("merge-utils")
+
+
+# ### Set Load/Save Paths
+# 
+
+# In[2]:
+
+
+# paths to load features/index from
+cp_features_save_path = pathlib.Path(
+    "/media/roshankern/63af2010-c376-459e-a56e-576b170133b6/data/cell-health-nuc-CP/"
+)
+dp_features_save_path = pathlib.Path(
+    "/media/roshankern/63af2010-c376-459e-a56e-576b170133b6/data/cell-health-nuc-DP/outputs/efn_pretrained/features"
+)
+dp_index_path = pathlib.Path(
+    "/media/roshankern/63af2010-c376-459e-a56e-576b170133b6/data/cell-health-nuc-DP/inputs/metadata/index.csv"
+)
+
+# path to save merged features to
+merged_features_save_path = pathlib.Path(
+    "/media/roshankern/63af2010-c376-459e-a56e-576b170133b6/data/cell-health-nuc-merged/"
+)
+merged_features_save_path.mkdir(parents=True, exist_ok=True)
+
+
+# ### Merge Features
+# 
+
+# In[3]:
+
+
+for cp_output_path in cp_features_save_path.iterdir():
+    plate = cp_output_path.name
+    print(f"Merging plate {plate} ...")
+
```

Comment:
consider provinding a general comment explaining what the for loop is doing just for readers to know what it is going to happen.